src/crypto/asmcrypto.js

```diff
@@ -0,0 +1,3 @@
+//# sourceMappingURL=asmcrypto.js.map
```

Comment:
I'm wondering if we can include this dependency as a lib using npm? If the author has not published a versioned bundle on the public npm repo we could either ask him to do so (which in my experience folks are happy to do), or we could just add a reference in package.json like so, as there is actually a release branch with git tags:

    "dependencies": {
        "asmcrypto.js": "https://github.com/vibornoff/asmcrypto.js/tarball/v0.0.8"
    }